src/components/ContextMenu.tsx

```diff
@@ -9,29 +9,33 @@ type ContextMenuOptionsType = {
   name: string;
   onClick: Function;
   args?: any[];
-  icon: Function;
+  icon?: Function | undefined;
 };
 
 type ContextMenuType = {
   options: ContextMenuOptionsType[];
-  showDotDotDot: boolean;
+  showEllipsis: boolean;
 };
 
 const styles: SXStyles = {
   container: {
     margin: '0',
     width: 'unset',
   },
-  menu: {
+  menuRelative: {
+    position: 'relative',
+  },
+  menuAbsolute: {
     display: 'flex',
     flexDirection: 'column',
     borderRadius: '8px',
     border: `1px solid ${theme.colors.borderColor}`,
     boxShadow: `0px 4px 40px rgba(0, 0, 0, 0.08)`,
     position: 'absolute',
-    zIndex: '11',
-    left: '150px',
+    zIndex: '100',
```

Comment:
no todos, just a reminder 
we need to make zIndex a constant in theme object so we can use constants. I didn't originally cuz I wanted to see what all layer we need.